server/app.py

```diff
@@ -1,10 +1,24 @@
 from flask import Flask
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
-
+from flask_sqlalchemy import SQLAlchemy
 
 app = Flask(__name__)
 
+#creates a postgres database connection with postgresql://username:password@localhost/dbname
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:1234@localhost/team-winter'
```

Comment:
can we make this a variable in .env and then gitignore the .env file

Reply:
If you are going to do this, you should write a quick readme on how to start the app with postgres